src/components/ChangePlan/Checkout/BillingInformation/BillingInformation.test.js

```diff
@@ -115,6 +144,15 @@ describe('BillingInformation component', () => {
     expect(selectCountry).toHaveValue(fakeBillingInformation.country);
     expect(selectProvince).toHaveValue(fakeBillingInformation.province);
     expect(inputPhone).toHaveValue(fakeBillingInformation.phone);
+
+    // form fields must not be disabled
+    expect(inputFirstName).not.toBeDisabled();
```

Comment:
// Assert 
Assert comment is missing from this and previous expect section, could you add it please?